plugins/remote-config/frontend/public/templates/conditions-drawer.html

```diff
@@ -1,23 +1,85 @@
 <cly-drawer
-    @submit="onSubmit"
-    @copy="onCopy"
-    :title="title"
-    :saveButtonLabel="saveButtonLabel"
-    v-bind="controls">
-    <template v-slot:default="drawerScope">
-        <cly-form-step id="conditions-drawer">
-            <cly-form-field :label="i18n('remote-config.condition-name')" name="condition_name" rules="required" v-slot:default>
-                <el-input
-                    v-model="drawerScope.editedObject.condition_name"
-                    :placeholder="i18n('placeholder.condition-name')">
-                </el-input>
-            </cly-form-field>
-            <cly-form-field :label="i18n('remote-config.seed-value')" name="seed_value" rules="required" v-slot:default>
-                <el-input
-                    v-model="drawerScope.editedObject.seed_value"
-                    :placeholder="i18n('placeholder.seed-value')">
-                </el-input>
-            </cly-form-field>
-        </cly-form-step>
-    </template>
+  @submit="onSubmit"
+  @copy="onCopy"
+  :title="title"
+  :saveButtonLabel="saveButtonLabel"
+  v-bind="controls">
+  <template v-slot:default="drawerScope">
+    <cly-form-step id="conditions-drawer">
+      <div class="cly-vue-drawer-step__section">
+        <div class="bu-pb-4">
+          <div class="font-weight-bold text-smallish cly-vue-remote-config-conditions-drawer__margin-bottom">
+            {{i18n('remote-config.condition-name')}}
+          </div>
+          <div class="color-cool-gray-50 text-small cly-vue-remote-config-conditions-drawer__margin-bottom">
+            {{i18n('placeholder.condition-name')}}
+          </div>
+          <validation-provider name="condition_name" rules="required" v-slot="v">
```

Comment:
please refrain from using validation-provider directly.
use `cly-form-field` instead.

Reply:
sure:)

Reply:
Incase you are worried that `cly-form-field` does not provide an option to add the description below the label.
you can add it to the component itself.
description can be used as a prop in the `cly-form-field` component then.

If you do this, you can also get rid of the class - `cly-vue-remote-config-conditions-drawer__margin-bottom`, and use bu-mb-2 or bu-mb-1 instead. Although i agree they don't equal 6px as mentioned in the design. We can discuss it with them.